tests/conftest.py

```
@@ -0,0 +1,129 @@
+    NormalCanAddressingInformation,
+    NormalFixedCanAddressingInformation,
+)
+from uds.utilities import RawBytesAlias, TransmissionDirection
+
+# Common
+
+
+@fixture(params=[
+    (0x00, 0xFF, 0xAA, 0x55),
+    [0x00],
+    bytearray(range(0x00, 0xFF)),
```

Ruff (0.12.2)

Unnecessary `start` argument in `range`

Remove `start` argument

(PIE808)